src/feature/hs/hs_dos.h

```diff
@@ -0,0 +1,44 @@
+/* Copyright (c) 2019, The Tor Project, Inc. */
+/* See LICENSE for licensing information */
+
+/**
+ * \file hs_dos.h
+ * \brief Header file containing denial of service defenses for the HS
+ *        subsystem for all versions.
+ **/
+
+#ifndef TOR_HS_DOS_H
+#define TOR_HS_DOS_H
+
+#include "core/or/or_circuit_st.h"
+
+#include "lib/evloop/token_bucket.h"
+
+#define HS_DOS_INTRODUCE_CELL_RATE_PER_SEC 25
```

Comment:
DOCDOCDOC here but I guess this is the most hanging part of the branch so there is not much to document yet.

Reply:
This is temporary. I'm not even sure that value is valid yet.

Reply:
I'd appreciate some more comments here regarding what 'rate' and 'burst' means in this context. Even a short one would be great.

Reply:
Tried to do that in fixup commit 143e36a09f

Reply:
Thanks!